remote_test.go

```diff
@@ -350,6 +350,28 @@ func TestClick(t *testing.T) {
 	}
 }
 
+func TestClick_Hidden(t *testing.T) {
+	t.Parallel()
+	wd := newRemote("TestClick_Hidden", t)
+	defer wd.Quit()
+
+	if err := wd.Get(serverURL); err != nil {
+		t.Fatal(err)
+	}
+	e, err := wd.FindElement(ByName, "hidden_name")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = e.Click()
+	if err == nil {
+		t.Fatal("expected clicking on hidden element to error")
+	}
+	want := "element not visible"
+	if err.Error() != want {
```

Comment:
We may want to consider catching this internally and throwing a Go error type (for use with a type assertion) instead of using string comparison.

But this is good enough for now, and I wouldn't want to block on that!